mealie/schema/group/group_shopping_list.py

```diff
@@ -100,14 +100,15 @@ class ShoppingListItemOut(ShoppingListItemBase):
     created_at: datetime | None = None
     update_at: datetime | None = None
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-
+    @model_validator(mode="after")
+    def post_validate(self):
         # if we're missing a label, but the food has a label, use that as the label
         if (not self.label) and (self.food and self.food.label):
             self.label = self.food.label
             self.label_id = self.label.id
 
+        return self
```

Comment:
Pydantic reassigns the instance for some reason, so yeah, we do actually have to `return self` here